SpatialGDK/Source/SpatialGDK/Public/Utils/RPCContainer.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Schema/RPCPayload.h"
+#include "Schema/UnrealObjectRef.h"
+#include "SpatialConstants.h"
+
+#include "CoreMinimal.h"
+
+struct FPendingRPCParams;
+using FPendingRPCParamsPtr = TUniquePtr<FPendingRPCParams>;
+DECLARE_DELEGATE_RetVal_OneParam(bool, FProcessRPCDelegate, const FPendingRPCParams&)
+
+struct FPendingRPCParams
+{
+	FPendingRPCParams(const FUnrealObjectRef& InTargetObjectRef, SpatialGDK::RPCPayload&& InPayload, int InReliableRPCIndex = 0);
+
+	// TODO: UNR-1653 Redesign bCheckRPCOrder Tests functionality
```

Comment:
This is directly part of this PR.

Reply:
It currently works as before with single worker, but we need to redesign it to support multiworker. I should probably update this comment and jira correspondingly.

Reply:
But this work shouldn't be separate work and we don't want to regress on multiworker.

Reply:
There is no resigned to be done, it just needs to be deleted.

Reply:
Having this index was a hacky solution to deal with commands not being ordered. And serves no function here, and will not be useful to us when we move the the RPCs that do work correctly in multiworker.